lib/tasks/readonly_database_user.rake

```diff
@@ -0,0 +1,15 @@
+namespace :db do
+  desc 'Create a readonly database user'
+  task create_readonly_user: :environment do
+    username = Figaro.env.database_readonly_username
+    password = Figaro.env.database_readonly_password
+
+    sql_statements = [
+      "CREATE USER #{username}",
```

Comment:
"CREATE USER #{username}  WITH ENCRYPTED PASSWORD '#{password}'"

Reply:
Maybe this too:
"GRANT SELECT ON ALL SEQUENCES IN SCHEMA public TO #{username}",